corehq/apps/app_manager/views/multimedia.py

```diff
@@ -27,3 +30,17 @@ def multimedia_ajax(request, domain, app_id):
         return render(request, "app_manager/partials/settings/multimedia_ajax.html", context)
     else:
         raise Http404()
+
+
+@require_deploy_apps
+@quickcache(['domain', 'app_id'], timeout=60 * 60)
+def get_multimedia_sizes(request, domain, app_id):
```

Comment:
docstring would be helpful

Reply:
https://github.com/dimagi/commcare-hq/pull/25864/commits/8785b87a1bb32c716341880dae47359ed7ea6c0a